package.json

```diff
@@ -4,8 +4,8 @@
   "author": "USWDS",
   "license": "CC0-1.0",
   "scripts": {
-    "federalist": "npm install && ./build.sh && npm run uswds-copy-assets && npm run uswds-build-sass && npm run build",
-    "init": "npm install && npm update uswds && npm run uswds-copy-assets && npm run uswds-copy-theme && npm run uswds-build-sass",
+    "federalist": "npm run init && npm run build && npm run uswds-build-sass",
+    "init": "./build.sh && npm run uswds-copy-fonts & npm run uswds-copy-img & npm run uswds-copy-js",
```

Comment:
We don't really have theme files any more post–USWDS 3 and it didn't seem like it was worth having a sync just to scaffold the sass entry point if we could just include it in the project?

Reply:
The build script needs to run before copying assets, since they would be copied from the `dist` directory

Reply:
Rewrote all parallel and sequential script using `npm-run-all`. I've swapped out `concurrently` since it's not always playing well with the watch script — and I like the simple syntax of `npm-run-all`